Assignment 5.Rmd

```diff
@@ -40,38 +50,84 @@ We want to see if we can build a decision tree to help teachers decide which stu
 
 #Take a look at our outcome variable "score"
 ```{r}
+
 hist(D1$score)
+
 ```
 
 #Create a categorical outcome variable based on student score to advise the teacher using an "ifelse" statement
 ```{r}
+
 D1$advice <- ifelse(D1$score <=0.4, "intervene", ifelse(D1$score > 0.4 & D1$score <=0.8, "monitor", "no action"))
+
 ```
 
 #Build a decision tree that predicts "advice" based on how many problems students have answered before, the percentage of those problems they got correct and how many hints they required
 ```{r}
+
 score_ctree <- ctree(factor(advice) ~ prior_prob_count + prior_percent_correct + hints, D1)
 ```
 
 #Plot tree
 ```{r}
+
 plot(score_ctree)
+
 ```
 
 Please interpret the tree, which two behaviors do you think the teacher should most closely pay attemtion to?
 
+######
+I would look at students that have asked for more than 12 hints or students that have gotten less than 63% of previous problems correct.
```

Comment:
I would add that you are offering this advice because the tree splits on >12 hints and >63% correct.